src/Surfnet/ServiceProviderDashboard/Infrastructure/Manage/Dto/MetaData.php

```diff
@@ -194,7 +194,10 @@ public function getSignatureMethod()
 
     public function getCertData()
     {
-        return $this->certData;
+        if ($this->certData) {
```

Comment:
Judging the way EngineBlock deals with certificates from manage, we might need to revise this a little bit. For example, EB is aware that multiple certificates might be in place. It is knowledgable on how to deal with them. I don't say we need to copy this logic. For now, using the primary certificate should do the trick!

What I do think is a valuable addition is to add a certificate validator/parser. Maybe the certificate of some other entity is set with the BEGIN/END 'envelope', this change would break that certificate.

For inspiration you might want to check: `\OpenConext\EngineBlock\Metadata\Entity\Assembler\PushMetadataAssembler::assembleCertificates`